common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
+// LMS Problem Feedback Revamp styling
+// Mirror styles from the Pattern Library
+
+// NOTE: This is the left-to-right (LTR) configured style compile.
```

Comment:
I would remove this comment, and make this file work for both directions. There isn't an RTL version...